scripts/sct_image.py

```diff
@@ -202,7 +202,9 @@ def main(args=None):
         im_out = None
 
     elif arguments.keep_vol is not None:
-        index_vol = arguments.keep_vol
+        index_vol = (arguments.keep_vol).split(",")
```

Comment:
I see that we use sometimes single quotes and sometimes double quotes.

For instance 
```python
    dim_list = ['x', 'y', 'z', 't']
```
But here,
```python
        index_vol = (arguments.keep_vol).split(",")
```
Would it be worth it to try to keep the same convention across the code?

Reply:
If that's the case, I think sticking to single quotes might be a good idea since when we use `argparse`, when defining our strings, we stick to having single quotes for strings and use double quotes within strings when we want to add examples. 

If ever it is worth  to assign a convention when single/double quotes in examples, I think it would be better to stick with single. However, I don't know if we want to assign a convention for something like this.

Thoughts?

Reply:
I think we should stick to single quotes. In my opinion standardizing the code is a good practice to keep it clean. Any thoughts @jcohenadad ?

Reply:
yes for convention and yes for single quote 👍

Reply:
Sounds good! Changed in commit [8bb3201](https://github.com/neuropoly/spinalcordtoolbox/commit/8bb320172a2485ce8815f257ee1c1f55bab3c0b4)